src/Provider/Vkontakte.php

```diff
@@ -104,33 +104,32 @@ public function getUserProfile()
 
         $data = new Collection($response->response[0]);
 
-        if (!$data->exists('id')) {
+        if (! $data->exists('id')) {
```

Comment:
please keep it as it was and do not use extra space for future `if` statements.

Reply:
@ApacheEx pls, publicate your code style conversation